amlb/benchmarks/openml.py

```diff
@@ -38,7 +42,8 @@ def load_oml_benchmark(benchmark: str) -> Tuple[str, Optional[str], List[Namespa
         data = openml.datasets.get_dataset(t.dataset_id, download_data=False, download_qualities=False)
         tasks = [Namespace(name=str_sanitize(data.name),
                            description=data.description,
-                           openml_task_id=t.id)]
+                           openml_task_id=t.id,
+                           id="{}/t/{}".format(domain, t.id))]
```

Comment:
```suggestion
                           id="{}.org/t/{}".format(domain, t.id))]
```
originally, for openml, we wanted to have the `id` == short url to the task/dataset, that's why in the validation, the task_id `42` gives a unique `id=openml.org/t/42`, making it easy for users to look at the task description,  and also for the `amlb_report` module to load metadata.
So it is actually quite important to keep this id as a proper url.

Reply:
Sorry, this was plain oversight.